apps/web-giddh/src/app/inventory/components/aside-pane/aside-pane.components.html

```diff
@@ -8,16 +8,16 @@ <h3>Create</h3>
     <button id="back" (click)="backButtonPressed()" *ngIf="hideFirstScreen">Back</button>
 
     <div class="aside-body flexy-child vmiddle asidePanelLedger " *ngIf="!hideFirstScreen" vr-item>
-        <div class="pd1 alC">
```

Comment:
what is alC class ??

Reply:
if you can see, alc class line is removed